publish-chartpress-images/Dockerfile

```diff
@@ -5,7 +5,8 @@ FROM docker:24.0.6-git
 RUN apk add python3 py-pip py3-ruamel.yaml && \
     wget -O /usr/bin/yq "https://github.com/mikefarah/yq/releases/download/3.1.1/yq_linux_amd64" && \
     chmod 0755 /usr/bin/yq && \
-    pip3 install -U pip chartpress==2.1.0 && \
+# requests 2.32.0 has a bug with not allowing docker+http protocol, see requests/issues/6707
+    pip3 install -U pip chartpress==2.1.0 docker<2.32.0 && \
```

Comment:
I thought requests was supposed to be at `<2.32.0` not docker as you have it here. Or are the versions on docker tracking the requests pacakge 1:1?